crates/napi/src/bindgen_runtime/module_register.rs

```diff
@@ -138,6 +160,57 @@ pub fn register_class(
   val.1.extend(props.into_iter());
 }
 
+#[inline]
+/// Get `JsFunction` from defined Rust `fn`
+/// ```rust
+/// #[napi]
+/// fn some_fn() -> u32 {
+///     1
+/// }
+///
+/// #[napi]
+/// fn return_some_fn() -> Result<JsFunction> {
+///     get_js_function(some_fn_js_function)
+/// }
+/// ```
+///
+/// ```js
+/// returnSomeFn()(); // 1
+/// ```
+///
+pub fn get_js_function(raw_fn: ExportRegisterCallback) -> Result<JsFunction> {
+  FN_REGISTER_MAP
+    .borrow_mut()
+    .get(&raw_fn)
+    .and_then(|(env, cb, name)| {
+      let mut function = ptr::null_mut();
+      let name_len = name.len() - 1;
+      let fn_name = unsafe { CStr::from_bytes_with_nul_unchecked(name.as_bytes()) };
+      check_status!(unsafe {
+        sys::napi_create_function(
+          *env,
+          fn_name.as_ptr(),
+          name_len,
+          *cb,
+          ptr::null_mut(),
+          &mut function,
+        )
+      })
+      .ok()?;
+      Some(JsFunction(Value {
+        env: *env,
+        value: function,
+        value_type: ValueType::Function,
+      }))
+    })
+    .ok_or_else(|| {
+      crate::Error::new(
+        crate::Status::InvalidArg,
+        "JavaScript function not existed".to_owned(),
```

Comment:
“JavaScript function <name> does not exists” reads better

Reply:
Thanks